### PR TITLE
OJ-18051: Chunk up long project_ids for GET request

### PR DESCRIPTION
**Pull request by zdwarren**
A customer is having a problem of the GET url being too long.
This ticket will chuck on the project_ids if they are too long.

Tested locally with ortho.

Jira Ticket:
https://jelly-ai.atlassian.net/browse/OJ-18051

### CODE REVIEW

File: jf_agent/jf_jira/jira_download.py
Comment (zdwarren) [inline jf_agent/jf_jira/jira_download.py:296]: This is the section with the changes - the diff is confusing because the for loop changed the spacing for everything below it.

**Review by eliwind — COMMENTED** [review]
This seem like a valid way to chunk up GET requests.  I'm curious though about GET for POST: there is another place (where we download the full issues rather than just the update timestamps) where we make the same requests with POST instead to avoid this issue.

Is there a reason this part ought to use GET / be different?

I'm not trying to block merging here, just curious to understand the thinking better.

**Comment by eliwind** [comment]
Also worth looking at the direct-connect version of this code too -- whatever we do in the agent we should probably do there as well.

This looks fine to me, I leave the relative merits of going to POST for consistency with the other place we make this call vs leaving it as a GET for less risk perturbing existing customers to your call.

Other thought: is there reason to think _think_ URL limit is the right one?  Will other load balances / HTTP servers / whatever else have a smaller limit (and thus maybe we should chunk even smaller?)

**Review by eliwind — APPROVED** [review]

**Review by mattklein — APPROVED** [review]

**Pull request merged**
